Avoid revealing secrets in wake and shutdown logs

netmikey opened the pull request:
Fixes #36

AlexGustafsson commented:
A heads up: after merging your other PR, this will now contain merge conflicts. Since the master branch now contains logging where you fixed the typos, git will want you to add the same changes to this PR. If you want further help with that or if my confusing explanation made the situation worse for you, just shout! :smile:

Pro tip:
Use a interactive rebase towards the master branch. This will make sure that the git history stays clean. Basically this means that your other changes will be applied before the changes that are currently in this PR. See https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase.

netmikey commented:
I found the self-documenting approach of an intermediate const assignment the most elegant, so I went for that.

I also rebased my branch on the current master.

I added [this check](https://github.com/AlexGustafsson/homebridge-wol/pull/56/files#diff-988817bb077e077e6d855ab16ff784a8R175) because calling `.replace(...)` on `this.config.shutdownCommand` when it isn't present would've caused an error at runtime. If I read your code correctly, [this log](https://github.com/AlexGustafsson/homebridge-wol/pull/56/files#diff-988817bb077e077e6d855ab16ff784a8L196) will be triggered when no `shutdownCommand` is configured, so _something_ would end up in the log in any case.

Let me know your thoughts 🙂

AlexGustafsson commented:
Merged!
